From 0c4b3b2226c3d85a6c421e414d1ebe2d171e9ee7 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 5 Jun 2014 12:37:49 -0400
Subject: [PATCH 68/72] [ovirt] add package list to ovirt plugin

Make sure that the oVirt plugin only runs if the ovirt-engine
package is present.

This will need to be expanded when the ovirt-engine-dwh and
ovirt-engine-reports support is merged (Issue #236 and Issue #237).

Signed-off-by: Brym M. Reeves <bmr@redhat.com>
---
 sos/plugins/ovirt.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sos/plugins/ovirt.py b/sos/plugins/ovirt.py
index acd6ad3..c6f8957 100644
--- a/sos/plugins/ovirt.py
+++ b/sos/plugins/ovirt.py
@@ -28,6 +28,8 @@ from sos.plugins import Plugin, RedHatPlugin
 class Ovirt(Plugin, RedHatPlugin):
     """oVirt Engine related information"""
 
+    packages = ('ovirt-engine',)
+
     DB_PASS_FILES = re.compile(
         flags=re.VERBOSE,
         pattern=r"""
-- 
1.9.3

